scss/components/_contact-items.scss

```diff
@@ -0,0 +1,39 @@
+// Contact items
+// List items on the contact page
+
+.contact-item {
+  @include clearfix();
+  margin-bottom: 4rem;
+}
+
+.contact-item--half {
+  @include media($med) {
+    @include span-columns(6);
+
+    &:nth-child(even) {
+      @include omega();
+    }
+  }
+}
+
+.contact-item__title {
+  margin-bottom: .5rem;
+}
+
+.contact-item__icon {
+  float: left;
+  margin-right: 2rem;
+}
+
+.contact-item__content {
+  float: left;
+}
+
+.contact-item__content--offset {
```

Comment:
Selector `contact-item__content--offset` should be written in lowercase with hyphens